src/cmd/mod.rs

```diff
@@ -1,10 +1,14 @@
 use std::process::exit;
 
 use console::Term;
+use serde::{Deserialize, Serialize};
 use tracing::{info, warn};
 use url::Url;
 
 use crate::git::has_staged_changes;
+use crate::recover::{
+    ask_for_recovery, clear_recovery_file, read_recovery_file, write_recovery_file,
+};
```

Comment:
i might rename the file to recovery.rs and call `recovery::ask` etc instead of doing this